testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/page/WelcomeScreen.java

```diff
@@ -41,21 +41,21 @@
     @FindBy(xpath = "//*[@id='" + ROOT_ELEMENT_ID + "']//header")
     private WelcomeScreenHeader header;
 
-    @FindBy(xpath = "//*[@id='landing-personal-info']/a")
+    @FindBy(xpath = "//a[@id='landing-personal-info']")
```

Comment:
Just out of curiosity, what's the reason to make this change?

Reply:
I'm also fixing a test in this PR